openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -93,6 +101,10 @@ def get_course_outline(course_key: CourseKey) -> CourseOutlineData:
 
     See the definition of CourseOutlineData for details about the data returned.
     """
+    # Record the course separately from the course_id usually done in views,
+    # to make sure we get useful Span information if we're invoked by things
+    # like management commands, where it may iterate through many courses.
+    set_custom_attribute('learning_sequences.api.course_id', str(course_key))
```

Comment:
I'm unclear whether you are referring to multiple calls within the same Transaction, or multiple Transactions that are coming in from a path that doesn't have a `course_id` custom attribute? Either way, I'm interested in seeing the data, but the comment could ultimately be more clear.  

If this were for multiple calls in the same Transaction, would this really result in additional spans, or would this custom attribute just clobber itself and you'd get the last one? This might be a moot question, if this isn't what you meant.